### PR TITLE
feat: support using Fiammetta to recover others' emotion

### PR DESCRIPTION
**Pull request by LingweiMeng**
We can use Fiammetta to recover other agents' emotion in a given room now.
The given room should be `room_x_y`
So its input format in the command line is similar to using drones.

And I also change some absolute positions into relative positions in `base_construct.py`.

[reference](https://www.bilibili.com/video/BV1mZ4y1z7wx)

### CODE REVIEW

**Comment by LingweiMeng** [comment]
If you do not have a account to test features of Fiammetta, please contact me.

**Comment by LingweiMeng** [comment]
> Oh, I forgot to add the image of Fia's Elite 2. I'll test and add it later.
> 
> BTW, if you do not have an account to test the features of Fiammetta, please contact me.

Have added Fiammetta Elite 2 images, and improved the robustness.

**Comment by Konano** [comment]
You don't need to merge the new commits on the dev branch, which can lead to confusion during review.

**Comment by Konano** [comment]
Can you force push the two commits `ceb7378700b9b2e67e57416318036bf8bd05c526` and `4c142cc0e1b5d44512604b4f31fa7110fa142079` again?

Rebase maybe can help you.

**Comment by LingweiMeng** [comment]
> Can you force push the two commits `ceb7378700b9b2e67e57416318036bf8bd05c526` and `4c142cc0e1b5d44512604b4f31fa7110fa142079` again?
> 
> Rebase maybe can help you.

Sorry for the confusion. And thx for reminding me of this.
Is this ok now?

File: README.md [hunk 1]
```diff
@@ -38,6 +38,7 @@
 - 自动收取邮件奖励
 - 自动收取并安放线索
 - 自动消耗无人机加速制造站或贸易站
+- 自动使用菲亚梅塔恢复指定房间心情最低干员的心情并重回岗位（工作位置不变以避免重新暖机） [[参考使用场景](https://www.bilibili.com/video/BV1mZ4y1z7wx)]
 - 自动更换基建排班干员（需要搭配配置文件使用）
```
Comment (Konano) [inline README.md:42]:
```suggestion
- 自动更换基建排班干员（需要搭配配置文件使用）
- 自动使用菲亚梅塔恢复指定房间心情最低干员的心情并重回岗位（工作位置不变以避免重新暖机） [[参考使用场景](https://www.bilibili.com/video/BV1mZ4y1z7wx)]
```

File: arknights_mower/solvers/base_construct.py [hunk 2]
```diff
@@ -636,6 +643,277 @@ def agent_arrange(self, plan: tp.BasePlan) -> None:
         logger.info('返回基建主界面')
         self.back()
 
+<<<<<<< HEAD
```
Comment (Konano) [inline arknights_mower/solvers/base_construct.py:646]: Unhandled conflicts when merging

**Review by Konano — CHANGES_REQUESTED** [review]

File: arknights_mower/solvers/base_construct.py (hunk 2 above)
Comment (LingweiMeng) [reply arknights_mower/solvers/base_construct.py]: Done. Thanks for your help.    :)

File: arknights_mower/solvers/base_construct.py [hunk 3]
```diff
@@ -636,6 +643,218 @@ def agent_arrange(self, plan: tp.BasePlan) -> None:
         logger.info('返回基建主界面')
         self.back()
 
+    def choose_agent_in_order(self, agent: list[str], exclude: list[str] = [], exclude_checked_in: bool = False, dormitory: bool = False):
```
Comment (Konano) [inline arknights_mower/solvers/base_construct.py:646]: 参数 `exclude` 的默认值最好是 None 而不是空 List，使用空 List 作为默认值会导致多次调用函数时共享同一个 List 变量。

更加详细的说明：https://www.bilibili.com/video/av894493876

**Review by Konano — APPROVED** [review]

File: arknights_mower/solvers/base_construct.py (hunk 3 above)
Comment (LingweiMeng) [reply arknights_mower/solvers/base_construct.py]: 谢谢指点

**Pull request merged**